server/routers/auth.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const path = require('path');
+require('dotenv').config({ path: path.join(__dirname, '../.env') });
+const { Types } = require('mongoose');
+const { User, Cart } = require('../models');
+const { signToken, verifyToken } = require('../jwt');
+
+const auth = express.Router();
+const { ObjectId } = Types;
+
+auth.post('/register', (request, response) => {
+  if (!request.body) {
+    return response.sendStatus(400);
+  }
+
+  const { email, password } = request.body;
+  const user = new User({
+    email,
+    password,
```

Comment:
@IgorChaikin Also storing plain password in database is anti-pattern, is case of data leak, password will be compromised. Best practice is to store hashed version of password in database and verify hash on login. I can recommend https://www.npmjs.com/package/bcryptjs library